cmd/wait/runner.go

```diff
@@ -74,6 +77,24 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		return microerror.Mask(err)
 	}
 
+	// Create k8s clients for the control plane
```

Comment:
charts are on the workload cluster, not the control plane?

Reply:
fixed